templates/data/opensearch/support.html

```diff
@@ -0,0 +1,544 @@
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h5 class="p-heading--5">Full-stack support</h5>
+        </div>
+        <p class="p-equal-height-row__item">
+          As the publisher of Ubuntu, Canonical uniquely supports the full stack. Complex issues in enterprise
+          data platforms require analysis and fixes all the way down the stack. Get a single solution for best in
+          class support &mdash; with no loose ends.
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item u-hide--medium">
+          <div class="p-image-container is-highlighted u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/ce91b0be-Rectangle 2-2.png",
+                        alt="",
+                        width="852",
+                        height="1279",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h5 class="p-heading--5">10 years of security</h5>
+        </div>
+        <p class="p-equal-height-row__item">
+          Not every organisation is willing or able to upgrade their critical data infrastructure on a regular
+          basis. With Canonical, your service stays secure for up to 10 years with no need for major upgrades.
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item u-hide--medium">
+          <div class="p-image-container is-highlighted u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/23432d1c-Rectangle 2-3.png",
+                        alt="",
+                        width="852",
+                        height="1279",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h5 class="p-heading--5">Fast response time</h5>
+        </div>
+        <p class="p-equal-height-row__item">
+          We guarantee 1 hour response time for the most critical issues. Our expertise underpins your Service
+          Level Agreement (SLA).
+        </p>
+      </div>
+    </div>
+    <hr class="is-muted" />
+    <div class=" u-align--right">
+      <a class="p-button--positive"
+         href="/contact-us"
+         aria-controls="data-opensearch-modal">Get
+      support</a>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row p-section--shallow">
+      <hr />
```

Comment:
Please choose to either put hr into the room or use is-fixed-width to keep it consistent